akka-actor-typed-tests/src/test/java/akka/actor/typed/javadsl/ActorLoggingTest.java

```diff
@@ -68,4 +69,20 @@ public void loggingProvidesMDC() {
     eventFilter.awaitDone(FiniteDuration.create(3, TimeUnit.SECONDS));
 
   }
+
+  @Test
```

Comment:
@johanandren I added a Java test, but when I experimented with parameters on the `CustomEventFilter` in this test and `loggingProvidesMDC` I observed it didn't work as expected.  When I changed the occurrences to `0` or `2`, the tests still passed.  My matching condition doesn't seem to be used either.

Reply:
Strange that we use the Scala eventfilter here and not the `akka.testkit.javadsl.EventFilter`. I think it'd be fine if you left it as expected and we create a separate issue to track switching over to the java event filter and verifying that it works as expected with the occurrencies.

Reply:
If I have some time I'll revisit this in another PR, if that's alright.

Reply:
Yes, absolutely, completely up to you!